### PR TITLE
storage-september-release

### PR DESCRIPTION
**Pull request by microzchang**
# Pull Request Checklist

Please leverage this checklist as a reminder to address commonly occurring feedback when submitting a pull request to make sure your PR can be reviewed quickly:

See the detailed list in the [contributing guide](https://github.com/Azure/azure-sdk-for-cpp/blob/main/CONTRIBUTING.md#pull-requests).

- [ ] [C++ Guidelines](https://azure.github.io/azure-sdk/cpp_introduction.html)
- [ ] Doxygen docs
- [ ] Unit tests
- [ ] No unwanted commits/changes
- [ ] Descriptive title/description
  - [ ] PR is single purpose
  - [ ] Related issue listed
- [ ] Comments in source
- [ ] No typos
- [ ] Update changelog
- [ ] Not work-in-progress
- [ ] External references or docs updated
- [ ] Self review of PR done
- [ ] Any breaking changes?

### CODE REVIEW

**Review by Jinming-Hu — APPROVED** [review]

**Review by antkmsft — APPROVED** [review]

File: sdk/storage/azure-storage-blobs/CHANGELOG.md [hunk 1]
```diff
@@ -1,14 +1,8 @@
...
+## 12.9.0 (2023-09-12)
 
-### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
-
-### Other Changes
+- Features in `12.9.0-beta.1` and `12.9.0-beta.2` are now generally available.
```
Comment (ahsonkhan) [inline sdk/storage/azure-storage-blobs/CHANGELOG.md:5]: We need to keep the heading structure in the changelog for our release notes tooling.

In this case, I recommend keeping this bullet under `### Features Added`

File: sdk/storage/azure-storage-files-datalake/CHANGELOG.md [hunk 2]
```diff
@@ -1,14 +1,8 @@
...
+## 12.8.0 (2023-09-12)
 
-### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
-
-### Other Changes
+- Features in `12.8.0-beta.1` and `12.8.0-beta.2` are now generally available.
```
Comment (ahsonkhan) [inline sdk/storage/azure-storage-files-datalake/CHANGELOG.md:5]: Same here, keep this under some sub-heading. If not `Features Added` (if you feel that isn't appropriate), then `Other Changes` might be a good alternative.

The previous betas did have new features, so `Features Added` makes sense.

File: sdk/storage/azure-storage-files-shares/CHANGELOG.md [hunk 3]
```diff
@@ -1,17 +1,15 @@
 # Release History
 
-## 12.7.0-beta.2 (Unreleased)
+## 12.7.0 (2023-09-12)
 
 ### Features Added
 
-### Breaking Changes
+- Features in `12.7.0-beta.1` and `12.7.0-beta.2` are now generally available.
```
Comment (ahsonkhan) [inline sdk/storage/azure-storage-files-shares/CHANGELOG.md:7]: This approach is good, keeping the heading.

File: sdk/core/azure-core-test/src/test_proxy_manager.cpp [hunk 4]
```diff
@@ -286,6 +286,7 @@ void TestProxyManager::SetProxySanitizer()
         "st",
         "se",
         "sig",
+        "sv",
```
Comment (ahsonkhan) [inline sdk/core/azure-core-test/src/test_proxy_manager.cpp:289]: What's motivating this core test change here? Please pull this change out into its own PR, rather than into the storage release PR.

File: sdk/storage/azure-storage-blobs/CHANGELOG.md (hunk 1 above)
Comment (ahsonkhan) [reply sdk/storage/azure-storage-blobs/CHANGELOG.md:5]: cc @antkmsft (fyi)

Comment (antkmsft) [reply sdk/storage/azure-storage-blobs/CHANGELOG.md:5]: BTW, in our releases for other libraries, instead of saying "Changes from beta are now generally available", we actually duplicate the entries from the beta releases, sometimes pulling only the high-level ones and putting less details. This way, customers who only consume our stable releases, are seeing meaningful changelogs.
Otherwise, given that most of the changes go through a beta release first, all changelogs for the stable releases are forever going to look like "changes from <previous beta> are now generally available".

File: sdk/core/azure-core-test/src/test_proxy_manager.cpp (hunk 4 above)
Comment (microzchang) [reply sdk/core/azure-core-test/src/test_proxy_manager.cpp:289]: Hi, since storage sdks will update sas version in each STG Release, so if the `sv` is not ignored, we have to record the test again in each release, this may cost a lot efforts. so this change is mainly to ignore `sv` query parameter. I've opened a PR [Storage/TestProxy playback mode ignore storage sas version](https://github.com/Azure/azure-sdk-for-cpp/pull/4941) for this.

**Review by Jinming-Hu — APPROVED** [review]

File: sdk/core/azure-core-test/src/test_proxy_manager.cpp (hunk 4 above)
Comment (ahsonkhan) [reply sdk/core/azure-core-test/src/test_proxy_manager.cpp:289]: Thanks, please revert the changes to this file from this PR.
